### ISSUE TITLE
Server with multiple IPs

### ISSUE THREAD

**Issue by anthoniob** [d3vilh/openvpn-server#13]
Hey, 
I wanted to know if there is a way to attach a specific IP to each client.
For example my server currently has 3 IPs
xxx.xxx.xxx.1
xxx.xxx.xxx.2
xxx.xxx.xxx.3

Therefore I want Client 1 and Client 2 to have attached ...xxx.1 ip address and Client 3 and 4 to the other ones. Is there a way to do it using the UI?

Thank you!

**Comment by d3vilh** [comment]
Hi @anthoniob,
with Openvpn-UI you can set the client IP only (IP which OpenVPN server will provide to the client internally).
This is standard OpenVPN Server configuration, so it is included in OpenVPN-UI as well.

Still you may have to use different "external server IPs" with help of iptables (NAT postrouting with MASQERADE to specific external server interface) with chain in [fw-rules.sh](https://github.com/d3vilh/openvpn-server/blob/fae80f5924d3ad9f5f5e50fbd72be4c5f840a011/fw-rules.sh). This needs to be tested though.

Sure, without UI it may be a headache to maintain whole the clients DB this way, so your request is legit but very rare.

If you can check and confirm exact rule/chain we need to have in `fw-rules.sh` then it is half of the solution, for the rest - I need to think how it is better to implement in interface, so this option will not annoy others.

But if you have just dozen clients to maintain then manual update of [fw-rules.sh](https://github.com/d3vilh/openvpn-server/blob/fae80f5924d3ad9f5f5e50fbd72be4c5f840a011/fw-rules.sh) can be the most feasible option.

**Comment by d3vilh** [comment]
this may work `iptables -t nat -A POSTROUTING -s 10.10.11.12 -o eth2 -j MASQUERADE` where `10.10.11.12` is specific openvpn static client IP and `eth2` is the interface which has another IP address.

**Comment by anthoniob** [comment]
Yes indeed the scale isn't that big(only dozens of clients) so I think the fw-rules.sh solution will be enough for me. Thank you for a quick response. I just wanted to make sure I'm not missing on a feature already built-in!

**Comment by anthoniob** [comment]
Hey @d3vilh,
I followed your advise and added two additional rules to the fw-rules.sh for two additional IPs I have assigned to the server.
The issue I encountered is that the openvpn client connects to the server but at the same time I can't access any website.
If I use the ip directly when looking up the website, I can establish a connection, otherwise not. For example 172.67.189.152 goes through and displays the error when connected to the vpn, but google.com doesn't work.

Here are the rules I've used.
iptables -t nat -A POSTROUTING -s 10.10.11.12 -o ens3:0 -j MASQUERADE
iptables -t nat -A POSTROUTING -s 10.0.10.1 -o ens3:1 -j MASQUERADE

Any help would be appreciated, thank you!
